robot/Cumulus/tests/browser/manage_households/add_address_managehh.robot

```diff
@@ -1,41 +1,70 @@
+
+*** Variables ***
+&{Address}       street=123 Dummy Street    city=Tracy, CA 99999        country=US
+
 *** Test Cases ***
 
-Add New Address to Household 
-    &{contact1} =  API Create Contact    MailingStreet=50 Fremont Street    MailingCity=San Francisco    MailingPostalCode=95320    MailingState=CA    MailingCountry=USA
-    Store Session Record    Account    &{contact1}[AccountId]
-    Go To Record Home  &{contact1}[AccountId]    
-    # Click Link    link=Show more actions
-    Click Link    link=Manage Household 
-    Wait For Locator    frame    Manage Household   
-    # Select Frame With Title   Manage Household
-    # Wait For Locator    span_button    Change Address
-    # Click Button    Change Address
-    Select Frame And Click Element    Manage Household    span_button    Change Address    
-    Click ManageHH Link     Enter a new address
-    Fill Address Form
-    ...                       Street=123 Dummy Street
-    ...                       City=Tracy
-    ...                       State=CA
-    ...                       Postal Code=99999
-    ...                       Country=US   
-    Click Span Button    Set Address
-    Click Button       title=Save
-    Unselect Frame
-    Go To Record Home    &{contact1}[Id]
-    Scroll Element Into View    text:Mailing Address
-    #Scroll Page To Location    0    1200
-    ${status}    Verify Details Address    Mailing Address    123 Dummy Street     Tracy, CA 99999     US
-    Should Be Equal as Strings    ${status}    pass
-    Go To Object Home          Account
-    Click Link    link=&{contact1}[LastName] Household
-    Select Tab  Details
-    Scroll Element Into View    text:Billing Address
-    #Scroll Page To Location    0    300
-    ${status}    Verify Details Address    Billing Address    123 Dummy Street     Tracy, CA 99999     US
-    Should Be Equal as Strings    ${status}    pass
+Add New Address to Household
+    [Documentation]                      Create a contact providing the address details using the backend API
+    ...                                  Navigate to the contact's Account details page and update the address
+    ...                                  To a new value from the manage housold page.
+    ...                                  Navigate to contact details and account details page.
+    ...                                  Verify the new address persists under both Mailing and Billing address details
+    ...                                  Verify Same address should be present on Account and Contacts
+    [tags]                               W-038348             feature: Manage Households
+
+    Go To Page                           Details
+    ...                                  Account
+    ...                                  object_id=&{contact1}[AccountId]
+
+    Click Link                           link=Manage Household
+    Go To Page                           Custom                             ManageHousehold
```

Comment:
I think Current page should be keyword would be more appropriate here as u are just making sure u are on manage household page before u do anything else

Reply:
changed to Current page